src/include/baselib/jni/JavaVirtualMachineConfig.h

```diff
@@ -36,8 +90,18 @@ namespace bl
         {
```

Comment:
I assume ultimately this object will also be populated from some JSON config file which can be used to configure the VM externally... ? but of course this is in future PRs